### PR TITLE
Improvements to classification metric docs

### PR DESCRIPTION
**Pull request by atavory**
Improvements to classification-metric UDF docs:

1. Better instructions using `CROSS JOIN UNNEST` instead of `MAP`
2. Code examples
3. Separate subsection to group related functions
Screen Shot 2019-08-10 at 23 08 34
Screen Shot 2019-08-10 at 23 08 45
Screen Shot 2019-08-10 at 23 08 54

### CODE REVIEW

**Comment by atavory** [comment]
@rschlussel Hi, split and rebased this, as you requested. Could you please review? Many thanks!

**Comment by atavory** [comment]
@rongrong Thanks for your comment on a typo in the docs - I've addressed it.

**Comment by atavory** [comment]
@rschlussel Could you please review this? Many thanks!

File: presto-docs/src/main/sphinx/functions/aggregate.rst [hunk 1]
```diff
@@ -329,121 +330,248 @@ Statistical Aggregate Functions
...
+    between 0 and 1.
 
-.. function:: var_pop(x) -> double
+    The thresholds are defined as a sequence whose :math:`j`-th entry is
 
-    Returns the population variance of all input values.
+    .. math ::
 
-.. function:: var_samp(x) -> double
+        t_j
```
Comment (rschlussel) [inline presto-docs/src/main/sphinx/functions/aggregate.rst]: maybe put this one inline, since it's just one variable.  What do you think?

Comment (atavory) [reply presto-docs/src/main/sphinx/functions/aggregate.rst]: @rschlussel You're right. It's better shortened. Thanks!

**Pull request merged**